configurations/eu_pcve_s04_pipeline.py

```diff
@@ -0,0 +1,280 @@
+                        analysis_location=AnalysisLocations.MOGADISHU_SUB_DISTRICT
+                    ),
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/somalia_district"),
+                        analysis_dataset="district",
+                        analysis_location=AnalysisLocations.SOMALIA_DISTRICT
+                    ),
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/somalia_region"),
+                        analysis_dataset="region",
+                        analysis_location=AnalysisLocations.SOMALIA_REGION
+                    ),
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/somalia_state"),
+                        analysis_dataset="state",
+                        analysis_location=AnalysisLocations.SOMALIA_STATE
+                    ),
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/somalia_zone"),
+                        analysis_dataset="zone",
+                        analysis_location=AnalysisLocations.SOMALIA_ZONE
+                    )
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["recently_displaced"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="recently_displaced",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/recently_displaced"),
+                        analysis_dataset="recently_displaced"
+                    )
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["household_language"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="household_language",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/household_language"),
+                        analysis_dataset="household_language"
+                    )
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["disability"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="disability",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/disability"),
+                        analysis_dataset="disability"
+                    )
+                ]
+            ),
+        ],
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+    ),
+    archive_configuration=ArchiveConfiguration(
+        archive_upload_bucket="gs://pipeline-execution-backup-archive",
+        bucket_dir_path="2022/EU-PCVE-S04"
```

Comment:
```suggestion
        bucket_dir_path="2023/EU-PCVE-S04"
```

Reply:
Thanks - have updated to this for consistency with the previous EU projects.
Note that there is now a minor inconsistency with the other projects though, which are usually organised by year then project then series rather than just by year and series.